apps/emqx/src/emqx_router.erl

```diff
@@ -767,6 +773,29 @@ detect_table_schema_vsn() ->
 is_empty(Tab) ->
     ets:first(Tab) =:= '$end_of_table'.
 
+mk_conflict_resolution_action(State) ->
+    NodesV1 = [Node || {Node, v1, _} <- State],
+    NodesUnknown = [Node || {Node, unknown, _} <- State],
+    Format =
+        "Following EMQX nodes are running with conflicting schema:"
+        "\n ~p"
+        "Please take the following steps to resolve the conflict:"
+        "\n 1. Stop listeners on those nodes: `$ emqx eval 'emqx_listener:stop()'`"
+        "\n 2. Wait until they are safe to restart."
+        "\n This could take some time, depending on the number of clients and their subscriptions."
+        "\n Those conditions should be true for each of the nodes in order to proceed:"
+        "\n   * `$ emqx eval 'ets:info(emqx_subscriber, size)'` prints `0`."
+        "\n   * `$ emqx ctl topics list` prints `No topics.`"
+        "\n 3. Upgrade the nodes to the latest version."
```

Comment:
Upgrade the node to version ~s

and ~s is replaced with current version string `emqx_release:version_with_prefix()`.